### ISSUE TITLE
Opening a website in the whitelist editor creates a duplicate

### ISSUE THREAD

**Issue by savchenko** [sereneblue/chameleon#471]
# Prerequisites
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

## Expected Behavior
Opening saved website in the Whitelist Editor opens the relevant record for editing.

## Current Behavior
Opening saved website in the Whitelist Editor creates a new record every time.

## Relevant settings
Tested with G-Suite admin console:
```json
{
        "id": "08dae130-b35a-4859-b847-84d6a599c0d5",
        "name": "admin.google.com",
        "sites": [
          {
            "domain": "admin.google.com"
          }
        ],
        "lang": "en-US",
        "profile": "default",
        "spoofIP": "",
        "options": {
          "name": false,
          "ref": false,
          "tz": false,
          "ws": false
        }
```

## Context (Environment)
Firefox v81.0.1
Microsoft Windows NT 10.0.19041.0

**Comment by sereneblue** [comment]
Hi @ savchenko,

When you go to the whitelist tab on Chameleon, does it say that the website is whitelisted?

**Comment by savchenko** [comment]
Yes, I can see the duplicates there.

**Comment by sereneblue** [comment]
@savchenko I'm able to reproduce the issue. It will be resolved in the next update.

**Comment by sereneblue** [comment]
@savchenko Please let me know if you're still experiencing issues with v0.21.5.

**Comment by savchenko** [comment]
Will do as soon as it's offered for download by Mozilla Addons (just checked, latest reported is v0.21.4.1)

![image](https://user-images.githubusercontent.com/300146/96665530-9e54e600-1344-11eb-9fe2-6004d656f690.png)

**Comment by sereneblue** [comment]
@savchenko You should be able to manually update by selecting `Check for updates` after clicking the gear icon on the addons page.

**Comment by savchenko** [comment]
Confirming as fixed in v0.21.5. Thanks!
